init-tools.cmd

```diff
@@ -1,6 +1,9 @@
 @echo off
 setlocal
 
+REM Workaround https://github.com/dotnet/coreclr/issues/2153
+set ComPlus_ReadyToRun=0
```

Comment:
> When you build leave node projects it will still run dotnet restore.

Are you saying when you msbuild a specific `.builds` or `.csproj`?  I did not try that.  I did, however, do an explicit `init-tools.cmd` followed by a `build.cmd` and that worked (e.g. no build errors).

I think that the root cause here is that csc that ships are part of the cli (not the runtime) has a ready to run image and we invoke that CSC during publish.  We didn't actually hit the problem during restore as far as I can tell.